chain-tx-enclave-next/mls/src/tree.rs

```diff
@@ -350,7 +390,148 @@ impl Tree {
         Ok(Self {
             nodes: vec![Node::Leaf(Some(creator_kp))],
             cs,
-            my_pos: 0,
+            my_pos: LeafSize(0),
         })
     }
+
+    /// spec: draft-ietf-mls-protocol.md#ratchet-tree-evolution
+    ///
+    /// update path secrets
+    ///
+    /// returns: direct path nodes, parent_hash, root_secret
+    pub fn encap(
```

Comment:
full word for `encap`

Reply:
or something different -- `Encap` has already a defined meaning in HPKE, but here it's something else

Reply:
maybe "evolve"